test/myMath.spec.js

```diff
@@ -1,12 +1,12 @@
 // Import chai.
-let chai = require('chai');
-let path = require('path');
+import chai from 'chai';
+import path from 'path';
 
 // Tell chai that we'll be using the "should" style assertions.
 chai.should();
 
 // Import the Math class.
-let MyMath = require(path.join(__dirname, '..', 'myMath'));
+import MyMath from '../MyMath';
```

Comment:
maybe because of old node version:
```
ubuntu@box673:~/es6-playground$ npm test

> es6-playground@1.0.0 test /home/ubuntu/es6-playground
> mocha --colors --compilers js:babel/register


module.js:340
    throw err;
          ^
Error: Cannot find module '../MyMath'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/home/ubuntu/es6-playground/test/myMath.spec.js:3:24)
    at Module._compile (module.js:456:26)
    at normalLoader (/home/ubuntu/es6-playground/node_modules/babel/node_modules/babel-core/lib/api/register/node.js:199:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/ubuntu/es6-playground/node_modules/babel/node_modules/babel-core/lib/api/register/node.js:216:7)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at /home/ubuntu/es6-playground/node_modules/mocha/lib/mocha.js:216:27
    at Array.forEach (native)
    at Mocha.loadFiles (/home/ubuntu/es6-playground/node_modules/mocha/lib/mocha.js:213:14)
    at Mocha.run (/home/ubuntu/es6-playground/node_modules/mocha/lib/mocha.js:453:10)
    at Object.<anonymous> (/home/ubuntu/es6-playground/node_modules/mocha/bin/_mocha:393:18)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
npm ERR! Test failed.  See above for more details.
ubuntu@box673:~/es6-playground$ ndoe --^C
ubuntu@box673:~/es6-playground$ npm --version
2.13.5
ubuntu@box673:~/es6-playground$ node --version
v0.10.33
```